lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -219,6 +219,15 @@
         }
       } // EditorHas.functionBar
 
+      if (ActiveOptions['section'] && $.markupSupportsEditableSections ( ActiveOptions.MarkupType )) {
+        var sectionLine = $.findSection(ActiveOptions['section'], LanguageDefinition.getDefinitionFor('gollum-helpers'));
+        if (sectionLine) {
+          window.ace_editor.gotoLine(sectionLine+1, 0, false);
```

Comment:
Is there any way to make this more expressive? Why `sectionLine+1`? What do the second and third argument mean? Perhaps just something like:
```javascript
window.ace_editor.gotoLine(sectionLine+1, offset=0, slap=false);
```

Reply:
Nice, didn't realize you could use variable assignments for this purpose!